subsys/net/ip/net_private.h

```diff
@@ -272,3 +272,20 @@ static inline void net_pkt_print_buffer_info(struct net_pkt *pkt, const char *st
 
 	printk("\n");
 }
+
+/* Only enable mutex lock/unlock debugging when needed as it will print
+ * lot of data which is not normally needed.
+ */
+#if 0
```

Comment:
Is it acceptable to have it disabled/enabled like this, or shall it rather be controlled by some new Kconfig option?

Reply:
These kind of if's are quite rare, I had one in `subsys/net/lib/websocket/websocket.c`. I am not sure we want to expose this to kconfig, I would imagine that this option would be enabled in very rare cases. Enabling the debugging permanently prints too much data without much benefit.

Reply:
Ok, understood 👍